### PR TITLE
add postgres 15 and rename existing postgres package to handle multip…

### PR DESCRIPTION
**Pull request by rawlingsj**
…le major version streams

Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

File: postgresql11.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package:
-  name: postgresql
+  name: postgresql11
```
Comment (rawlingsj) [inline postgresql11.yaml:2]: `postgresql11` or `postgresql-11 `?

Comment (amouat) [reply postgresql11.yaml:2]: Personally I like the "-" but I wonder if there's a reason Alpine don't have it...

Comment (amouat) [reply postgresql11.yaml:2]: debian (apt) seems to use a -

Comment (kaniini) [reply postgresql11.yaml:2]: We can use `-` or no dash.  `@` is restricted for tagged repositories.

Comment (rawlingsj) [reply postgresql11.yaml:2]: Do one of you two want to make a call as I don't have an opinion?  Then I'll leave or change this PR.  

I've got a followup for Java ready so wont push that until I know if we are using a dash or not.

**Review by kaniini — APPROVED** [review]

**Pull request merged**